File: android/src/main/kotlin/uz/udevs/udevs_video_player/models/TvProgram.kt
package uz.udevs.udevs_video_player.models

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class TvProgram(
    @SerializedName("scheduledTime")
    val scheduledTime: String,
    @SerializedName("programTitle")
    val programTitle: String,
    @SerializedName("isAvailable")
    val isAvailable: Boolean,
) : Serializable

File: lib/models/tv_program.dart
class TvProgram {
  String scheduledTime;
  String programTitle;
  bool isAvailable;

  Map<String, dynamic> toJson() {
    var map = <String, dynamic>{};
    map['scheduledTime'] = scheduledTime;
    map['programTitle'] = programTitle;
    map['isAvailable'] = isAvailable;
    return map;
  }

  @override
  String toString() {
    return 'TvProgram{scheduledTime: $scheduledTime, programTitle: $programTitle}';
  }

  TvProgram({
    required this.scheduledTime,
    required this.programTitle,
    required this.isAvailable,
  });
}
